cristin-person/src/test/java/no/unit/nva/cristin/person/model/cristin/CristinPersonTest.java

```diff
@@ -0,0 +1,71 @@
+package no.unit.nva.cristin.person.model.cristin;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.nio.file.Path;
+import no.unit.nva.cristin.person.Constants;
+import no.unit.nva.cristin.person.model.nva.Person;
+import nva.commons.core.JsonUtils;
+import nva.commons.core.ioutils.IoUtils;
+import org.junit.jupiter.api.Test;
+
+public class CristinPersonTest {
+
+    private static final ObjectMapper OBJECT_MAPPER = JsonUtils.dtoObjectMapper;
+    private static final String CRISTIN_GET_PERSON_JSON =
+        "cristinGetPersonResponse.json";
+    private static final String NVA_API_GET_PERSON_JSON =
+        "nvaApiGetPersonResponse.json";
+
+    @Test
+    void cristinModelBuildsCorrectlyWhenDeserializingPersonJson() throws IOException {
+        String body = getBodyFromResource(CRISTIN_GET_PERSON_JSON);
+        CristinPerson cristinPerson = fromJson(body, CristinPerson.class);
+        JsonNode node = OBJECT_MAPPER.readTree(body);
+
+        assertEquals(cristinPerson.getCristinPersonId(), node.get("cristin_person_id").asText());
```

Comment:
assertEquals(expected,actual). Although the order does not matter when the test succeeds, it matters when it fails. This is why I usually prefer the more intuitive assertThat(actual,is(equalTo(expected)))